### PR TITLE
Add cheatmd support

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: test/fixtures/cheatsheets.cheatmd [hunk 1]
```diff
@@ -0,0 +1,763 @@
+---
+title: Elixir
+category: Elixir
+layout: 2017/sheet
+tags: [New]
+updated: 2018-07-04
+weight: -10
+---
```
Comment (yordis) [inline test/fixtures/cheatsheets.cheatmd:8]: Do we suppose to do something with this?

Comment (josevalim) [reply test/fixtures/cheatsheets.cheatmd:8]: No, we won't have preludes I think. We can discard it :)

File: test/fixtures/cheatsheets.cheatmd [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+  def extensions, do: ["json"]
+end
+```
+
+See: [Module](http://elixir-lang.org/docs/stable/elixir/Module.html)
+
+## References
+{: .-one-column}
+
+- [Learn Elixir in Y minutes](https://learnxinyminutes.com/docs/elixir/)
```
Comment (josevalim) [inline test/fixtures/cheatsheets.cheatmd:754]: I think we can reduce the size of this cheatsheet. :)

File: lib/ex_doc/formatter/html.ex [hunk 3]
```diff
@@ -543,4 +552,33 @@ defmodule ExDoc.Formatter.HTML do
         {base, opts[:filename] || text_to_id(Path.rootname(base))}
     end)
   end
+
+  defp sectionize(list), do: sectionize(list, [])
```
Comment (josevalim) [inline lib/ex_doc/formatter/html.ex:556]: Let's move this function to the `Markdown` module, WDYT?

File: test/fixtures/cheatsheets.cheatmd (hunk 2 above)
Comment (josevalim) [reply test/fixtures/cheatsheets.cheatmd:754]: I think we can keep it small mostly so it is easier for us to test it. :)

File: lib/ex_doc/formatter/html.ex (hunk 3 above)
Comment (yordis) [reply lib/ex_doc/formatter/html.ex]: I thought on doing that, wasn't sure about the intention of such a module.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**